### ISSUE TITLE
Midpoint Review - Accessibility Feedback - MHV Medical Records, Phase 1 Download Records

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#97729]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Overall this looks good!
- I really appreciate the annotations on the sample PDFs!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Coding note:** If it's not already on your radar, be thinking about focus management for the alerts after the download starts. I would recommend sending focus to the alert when you land back on the "Download your medical records reports" page, so that the alert is immediately announced to screen reader users and they know that their action has succeeded. Alternatively, you could not do focus management and instead use an `aria-live` region around the alert --- which would be fine, but I think ultimately a little more cumbersome than moving focus.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
